openreview/venue/process/submission_revision_process.py

```diff
@@ -9,6 +9,7 @@ def process(client, edit, invitation):
     submission_name = domain.content['submission_name']['value']
 
     submission = client.get_note(edit.note.id)
+    print(edit.id)
```

Comment:
should we distinguish between a revision that deletes the note and another one that edits it?

otherwise the other authors won't know the paper was deleted.